src/components/Details.js

```diff
@@ -55,10 +55,12 @@ const Details = () => {
                       Key <span className='def-color'>Statistics</span> 
                     </h3>
                     <div className='text-center p-2'>
-                      <div className='mb-4 mx-3 market-row p-3 shadow text-success'>
+
+                    <div className='mb-4 mx-3 market-row p-3 shadow text-success'>
```

Comment:
- The design of the page is awesome👍

Reply:
Thanks so much ma'am🕺🏽🥳💃, make me happy,